### PR TITLE
Use Std_New for startup document creation

### PR DESCRIPTION
**Pull request by adrianinsaval**
This makes them use a translated string for the new document name
https://forum.freecadweb.org/viewtopic.php?f=10&t=69859&p=606617
@Kuzma30

### CODE REVIEW

**Comment by Kuzma30** [comment]
Test.
It work when I use New command from toolbar.
When I use "Create new..." from Start WB I get untranslated "Unnamed"

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7119">pipeline status</a> for feature branch [PR_7119](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7119). Pipeline [578985559](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/578985559) was triggered at [7705ecf](https://github.com/FreeCAD/FreeCAD/pull/7119/commits/7705ecf00ce433211ba4c73f4b57bdad73b2c93c). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by yorikvanhavre** [comment]
I'm not sure about this one... Running a command during the GUI startup seems risky and could lead to race conditions. Inside the start WB, that is okay, though.

**Comment by adrianinsaval** [comment]
I used the command to avoid code duplication but it's possible to just use the same code as the command in there if it's risky to use the command.

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7119">pipeline status</a> for feature branch [PR_7119](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7119). Pipeline [580656092](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/580656092) was triggered at [cefa547](https://github.com/FreeCAD/FreeCAD/pull/7119/commits/cefa547a9b3c594f6eac8639bf21b4a6e2c061c1). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Gui/MainWindow.cpp [hunk 1]
```diff
@@ -1276,9 +1276,7 @@ void MainWindow::delayedStartup()
     ParameterGrp::handle hGrp = WindowParameter::getDefaultParameter()->GetGroup("Document");
     if (hGrp->GetBool("CreateNewDoc", false)) {
         if (App::GetApplication().getDocuments().size()==0){
-            App::GetApplication().newDocument();
-            Gui::Command::doCommand(Gui::Command::Gui,
-                "Gui.activeDocument().activeView().viewDefaultOrientation()");
+            Application::Instance->commandManager().runCommandByName("Std_New");
```
Comment (adrianinsaval) [inline src/Gui/MainWindow.cpp:1279]: @wwmayer is there any risk in running a Gui command here?

**Comment by yorikvanhavre** [comment]
It's a design decision way back at the beginnings of FreeCAD... Basically App should work fully from Python, untranslated, and free from any Gui / Qt or localized elements. All the localization and translation should be done in Gui, so the App part would behave 100% identical, no matter the platform it runs on.

Of course later on this has been flexibilized a bit, there is some QtCore stuff in it, but I think the general idea is still valid.

**Comment by adrianinsaval** [comment]
was that not done because at the time qt was only had gui components? Does that logic apply to this case?

File: src/Gui/MainWindow.cpp (hunk 1 above)
Comment (wwmayer) [reply src/Gui/MainWindow.cpp:1279]: Should be fine.

**Comment by chennes** [comment]
@adrianinsaval I see this is marked as Draft -- is it actually still a work-in-progress?

**Comment by adrianinsaval** [comment]
I'm not sure how we left this one, I'll have to dive back in (but can't at the momment).
kuzma had made a different PR that also tackled this and it wasn't clear what was the approach we should really take for this. There was a question whether we could have translated stuff at the App level or not.

**Comment by chennes** [comment]
No rush, I'm just reviewing these older PRs to clean things up.

**Comment by adrianinsaval** [comment]
I highly doubt I'll do more work on this subject so better to just merge this one improvement than to remain with none of it. Recently the use of Std_New on startpage was merged from another PR so there's one less commit here. I don't know if @Kuzma30 did any more work on this subject

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7119">pipeline status</a> for feature branch [PR_7119](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7119). Pipeline [668085467](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/668085467) was triggered at [769d3ab](https://github.com/FreeCAD/FreeCAD/pull/7119/commits/769d3ab53782c80177c26650685cbdcc5c2e6f77). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7119">pipeline status</a> for feature branch [PR_7119](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7119). Pipeline [672827448](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/672827448) was triggered at [1fc9d26](https://github.com/FreeCAD/FreeCAD/pull/7119/commits/1fc9d2641f257d454816d369de3fd88a469f2599). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Pull request merged**